pkg/rpcclient/wsclient.go

```diff
@@ -808,7 +808,7 @@ func (c *WSClient) ReceiveExecutions(flt *neorpc.ExecutionFilter, rcvr chan<- *s
 // where sender corresponds to notary request sender (the second fallback transaction
 // signer) and signer corresponds to main transaction signers. nil value doesn't add
 // any filter. See WSClient comments for generic Receive* behaviour details.
-func (c *WSClient) ReceiveNotaryRequests(flt *neorpc.TxFilter, rcvr chan<- *result.NotaryRequestEvent) (string, error) {
+func (c *WSClient) ReceiveNotaryRequests(flt *neorpc.NotaryRequestFilter, rcvr chan<- *result.NotaryRequestEvent) (string, error) {
```

Comment:
@roman-khimov, are we worried about backwards compatibility here? I know that we put a lot of efforts previously for keeping the deprecated subscriptions API, but maybe we can make this change directly without marking this method as deprecated?